assets/src/dashboard/app/api/useTemplateApi.js

```diff
@@ -308,20 +308,20 @@ const useTemplateApi = (dataAdapter, config) => {
     [dataAdapter, templateApi]
   );
 
-  const fetchRelatedTemplates = useCallback(() => {
-    if (!state.templates) {
-      return [];
-    }
-    // this will return anywhere between 1 and 5 "related" templates
-    const randomStartingIndex = Math.floor(
-      Math.random() * state.templatesOrderById.length
-    );
-    return [...state.templatesOrderById]
-      .splice(randomStartingIndex, 5)
-      .map((id) => {
-        return state.templates[id];
-      });
-  }, [state.templatesOrderById, state.templates]);
+  const fetchRelatedTemplates = useCallback(
+    (currentTemplateId) => {
+      if (!state.templates || !currentTemplateId) {
+        return [];
+      }
+
+      return state.templatesOrderById
+        .filter((id) => id !== currentTemplateId) // Filter out the current/active template
+        .sort(() => 0.5 - Math.random()) // Randomly sort the array of ids
```

Comment:
Randomness is not good when we want to do Percy snapshots

Reply:
At the moment, at least to my knowledge, there aren't Percy snapshots of the Template Details view probably precisely because of this.  If we want to add Percy snapshots for this view we need to either properly implement Related Templates or hide them behind a feature flag.  

Currently, Related Templates are random and is also causing CI/CD tests to fail since it's possible for the active/selected template to display as a related template. This PR is mainly to resolve CI/CD problems for folks.